tests/positive/files/file.go

```diff
@@ -65,6 +65,40 @@ func CreateFileOnRoot() types.Test {
 	}
 }
 
+func CreateFileOnRootFromBase64() types.Test {
+	name := "Create Files on the Root Filesystem from base64 data"
```

Comment:
These aren't arbitrary strings anymore.  Maybe `files.create.base64`?

Reply:
Good catch, should have continued editing the file instead of trying to port the commit.